client/src/js/modules/core/components/Header.tsx

```diff
@@ -58,6 +58,11 @@ interface HeaderProps {
 const Header: React.SFC<HeaderProps> = ({ classes }) => {
   return (
     <div className={classes.Header}>
+      <div className={classes.links}>
+      <Link className={classes.button} to="/about">
```

Comment:
I think we should keep it to one button on a page, namely, the Add Your School button. The Add Your School button should be a [call to action](https://yoast.com/call-to-action/) and relatively visible (i.e. a bright color). The about link is less important, so #2b2d42 is a good choice for that, but I don't think we should have the border around it.  A simple large link should suffice